weed/command/scaffold.go

```diff
@@ -102,6 +102,10 @@ enabled = false
 dir = "./filerrdb"					# directory to store rocksdb files
 
 [mysql]  # or memsql, tidb
+# Empty insertQuery will use the default insert query
+# example insertQuery can be for UPSERT syntax:
+# insertQuery = """INSERT INTO ` + "`%s`" + ` (dirhash,name,directory,meta) VALUES(?,?,?,?) ON DUPLICATE KEY UPDATE meta = VALUES(meta)"""
```

Comment:
The example seems wrong. ``` ` + "`%s`" + ` ``` should not need the plus operator for the string concatenation.

Reply:
that is because the scaffold uses ` and MySQL uses ` to quote the table name, but go uses it as string literal...

Reply:
@chrislusf  already exists and working in:
https://github.com/chrislusf/seaweedfs/blob/96c62bd34d4d2c99dea3164a9c062abaddb4eaab/weed/command/scaffold.go#L135